core/java/iesi-core/src/main/java/io/metadew/iesi/script/execution/ExecutionRuntime.java

```diff
@@ -85,7 +85,7 @@ public ExecutionRuntime(ExecutionControl executionControl, String runId) {
         // Initialize extensions
 
         // Initialize data instructions
-        dataInstructions = DataInstructionRepository.getRepository(new GenerationObjectExecution());
+        dataInstructions = DataInstructionRepository.getRepository(new GenerationObjectExecution(),this);
```

Comment:
code formatting. you can auto-format the code using IntelliJ.
Normally the Sonarlint plugin will also complain about this.